index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" media="screen" href="css/style.css">
+
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <h1>"hello world, we are software developers!"</h1>
+    <!--
+    Hello Camper!
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+    - The freeCodeCamp Team
+    -->
+    <section id="Javascript_and_Java" class="main-section">
+      <h2>Javascript and Java</h2>
+       <nav id="navbar"><h3>It is a project</h3><a class="nav-link" href="#Javascript_and_Java"></a><div>js</div></nav>
+      <code></code>
+      <code></code>
+    </section>
+    <section id="Cats_and_Dogs" class="main-section">
+      <h3>Cats and Dogs</h3>
+      <div><h3>It is a project</h3><a class="nav-link" href="#Cats_and_Dogs"></a><div>dogs and cats</div></div>
+      <code></code>
+      <ul>
+        <li></li>
+        <li></li>
+        <li></li>
+        <li></li>
+        <li></li>
+      </ul>
+    </section>
+    <section id="Cell_Phones" class="main-section">
+      <h4>Cell Phones</h4>
+      <p></p>
+      <p></p>
+      <p></p>
+      <div><h3>It is a project</h3><a class="nav-link" href="#Cell_Phones"></a><div>samsung</div></div>
+      <code></code>
+    </section>
```

Comment:
Kindly add some proper CSS classes inside the tags in order to upgrade the styles in your project.

Reply:
- [ ] [OPTIONAL] Good job can and try using a couple of emojis and/or gifs in the review to make the code review more human-friendly. :smile_cat: